spec/passkey-server.yaml

```diff
@@ -414,6 +487,12 @@ components:
           schema:
             type: object
             properties:
+              type:
+                type:
+                  - string
+                  - 'null'
+                examples:
+                  - 'http://link-to-documentation'
```

Comment:
This property was removed and is not reintroduced, so it should be removed.

Reply:
Fixed